src/components/VBackToSearchResultsLink.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <!-- @todo: Separate the absolute container from the link itself. -->
   <VLink
-    class="flex flex-row items-center px-2 py-3 text-xs font-semibold text-dark-charcoal md:px-6 md:pt-4 md:pb-2 md:text-sr"
-    :href="path"
+    class="time inline-flex flex-row items-center gap-2 rounded-sm p-2 text-xs font-semibold text-dark-charcoal-70 pe-3 hover:text-dark-charcoal"
```

Comment:
What is the "time" CSS class?

Reply:
That's the name of the font styled used by @panchovm in the Figma mockups. If I had to guess, it started as the font-style used in the audio timestamps but then found use in other places without updating the name.

Reply:
I think you are referring to the [Time pill component](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=2484%3A22545&t=rP5ZmCU9JdBEliSA-4). I am seeing the Design Library and that is the only component named like that and used in all audio component's variants.

<img width="400" alt="CleanShot 2022-11-22 at 10 12 20@2x" src="https://user-images.githubusercontent.com/895819/203273487-634f4f2a-5489-4d35-bd05-3650f5352c93.png">

Reply:
I was referring to this:
![Screenshot 2022-11-22 at 8 01 29 PM](https://user-images.githubusercontent.com/16580576/203362243-8224070a-081b-4903-8e64-5542cc912a97.png)

Reply:
Oh, I got it. And yes, it was named time as it was created for the time pill.